waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/security/AccessControlHandlerFactory.java

```diff
@@ -36,36 +42,39 @@ public AccessControlHandlerFactory(FederationService federationService) {
     this.federationService = federationService;
   }
 
+  private static final Logger LOG = LoggerFactory.getLogger(AccessControlHandlerFactory.class);
+
   public AccessControlHandler newInstance(AbstractMetaStore federatedMetaStore) {
     switch (federatedMetaStore.getAccessControlType()) {
     case READ_ONLY:
+      LOG.debug("Creating {} accessControlHandler for {}", READ_ONLY, federatedMetaStore.toString());
```

Comment:
all these debug statements seem fairly superfluous and unnecessary, is it meaningful in the logs or can we just get rid of them

Reply:
This actually did help me in the debugging process, I don't think they hurt and can easily make visible a configuration mistake

Reply:
But they just print that READ_ONLY == READ_ONLY. 
Just add one debug line in the beginning of the method if you want to print that you're creating an accessControlHandler for a metastore.